index.js

```diff
@@ -45,6 +46,20 @@ function handleArgs(cmd, args, opts) {
 	};
 }
 
+function handleInput(spawned, opts) {
+	var input = opts.input;
+
+	if (input === null || input === undefined) {
+		return;
+	}
+
+	if (isStream(input)) {
+		input.pipe(spawned.stdin);
+	} else {
+		spawned.stdin.end(input);
+	}
```

Comment:
Hmm. Is there value to that though? I am all for preferring `return` over nesting some huge else block, but for a short one liner, I think `if`/`else` is more readable.

Depending on what we decide, we could make XO catch this one pretty easy.